react/features/lobby/components/AbstractLobbyScreen.js

```diff
@@ -0,0 +1,317 @@
+// @flow
+// eslint-disable-next-line no-unused-vars
```

Comment:
why is this needed?

Reply:
The linter doesn't seem to understand the `<> </>` tags and warns about the unused variable. But also warns if React is not present and we use react rendering. Seems like a linter bug.